tests/run-make/crate-loading/rmake.rs

```diff
@@ -0,0 +1,30 @@
+//@ only-linux
+//@ ignore-wasm32
```

Comment:
Question: does this only fail on wasm32, or does it also fail on wasm 64 targets? We could run `test-various` and `x86_64-gnu-llvm-17` try jobs to double check.